src/pages/invoices/common/hooks/useResolveTotalVariable.tsx

```diff
@@ -40,7 +40,7 @@ export function useResolveTotalVariable(props: Props) {
   const handleChange = (property: keyof ProductTableResource, value: unknown) =>
     props.onChange(property, value);
 
-  const resolveTranslation = useResolveTranslation();
+  const resolveTranslation = useResolveTranslation({});
```

Comment:
This is a big `oof` moment. Can we rather make the whole object optional?

Reply:
Sure, changed!